packages/nextjs/components/project/ProjectHeader.tsx

```diff
@@ -0,0 +1,256 @@
+      name: "L2BEAT",
+
+      image: "/assets/gradient-bg.png",
+      op: 15416,
+    },
+    {
+      name: "Polynya",
+
+      image: "/assets/gradient-bg.png",
+      op: 12416,
+    },
+    {
+      name: "DefiLlama",
+
+      image: "/assets/gradient-bg.png",
+      op: 20416,
+    },
+    {
+      name: "L2BEAT",
+
+      image: "/assets/gradient-bg.png",
+      op: 15416,
+    },
+    {
+      name: "Polynya",
+
+      image: "/assets/gradient-bg.png",
+      op: 12416,
+    },
+  ];
+
+  const handleAddBallot = () => {
+    setLoadingMessage("Adding to ballot");
+    setSuccessMessage("Selection added successfully");
+    setAddBallot(false);
+    setIsLoading(true);
+    setAddVote(false);
+    setVoteAmount("35,416");
+    setTimeout(() => {
+      // Spoofed API request to add to ballot
+      setIsLoading(false);
+      setIsSuccess(true);
+      setTimeout(() => {
+        // Spoofed response from api
+        setIsSuccess(false);
+      }, 2000);
+    }, 1000);
+  };
+
+  const handleSaveBallot = () => {
+    setLoadingMessage("Saving distribution");
+    setSuccessMessage("Distribution changed successfully");
+    setEditBallot(false);
+    setIsLoading(true);
+    setTimeout(() => {
+      // Spoofed API request to save ballot
+      setIsLoading(false);
+      setIsSuccess(true);
+      setTimeout(() => {
+        // Spoofed response from api
+        setIsSuccess(false);
+      }, 2000);
+    }, 1000);
+  };
+
+  const handleAddOrEditModal = (close: boolean, edit = false) => {
+    setEditBallot(!close && edit);
+    setAddBallot(!close && !edit);
+  };
+
+  return (
+    <div className="mx-auto">
+      <div className="max-w-full mx-auto">
+        <Image
+          width={1000}
+          height={1000}
+          src="/assets/project/image 3.png"
+          alt="project image"
+          className="block w-full"
+        />
+      </div>
+
+      <div className="flex justify-between">
+        <div className="ml-[30px] flex items-center flex-wrap">
+          <Image
+            width={500}
+            height={500}
+            className=" mt-[-20px] w-[134px] h-[134px] rounded-lg shadow-md  "
+            src="/assets/project/image 4.png"
+            alt="project image"
+          />
+          <div className="ml-[20px] mt-[-50px] flex-wrap">
+            <h1 className="pt-20 font-semibold sm:text-xl md:text-2xl  lg:text-4xl  leading-11">Orbiter Finance</h1>
+            <div className="flex justify-between flex-wrap">
+              <>
+                <div className="flex items-center gap-8 flex-wrap">
+                  <a
+                    href="#"
+                    className="bg-gradient-to-r from-[#FF0420] to-[#9104FF] inline-block text-transparent bg-clip-text"
+                  >
+                    @orbiter_finance
+                  </a>
+
+                  {addressCopied ? (
+                    <CheckCircleIcon
+                      className="text-xl font-normal text-sky-600 h-5 w-5 cursor-pointer"
```

Comment:
When the address is copied, the elements in this row jump slightly. can be fixed by adding ml-[6px] to CheckCircleIcon.